### DISCUSSION TITLE
How  to setup completely offline setup with hasura ?

### DISCUSSION THREAD

**Discussion by HariharanUmapathi** [hasura/graphql-engine#8965] [Question]
**What I Have ?**
Postgres 13 Installed on a windows machine
Hasura cli 

** What I need **

Need to access my database via hasura console offline 

![image](https://user-images.githubusercontent.com/62648127/189963259-4b7cf840-18a5-46b8-ae93-df3019fc6fe5.png)

While I'm trying to connect my local datatabase with hasura console

**Comment by adas98012** [comment]
Please try using the Database URL or Environment Variable (recommended) options.
Let us know if you still see the connection issue.
Also, consider using the newer versions of Hasura GraphQL (e.g. 2.11.0)

**Reply by HariharanUmapathi** [reply]
I have updated hasura cli version 2.11.1 today and tried with Database URL

 {
    "internal": [
        {
            "definition": "My postgres local",
            "reason": "Inconsistent object: connection error",
            "name": "source My postgres local",
            "type": "source",
            "message": "could not connect to server: Connection refused\n\tIs the server running on host \"localhost\" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5432?\ncould not connect to server: Cannot assign requested address\n\tIs the server running on host \"localhost\" (::1) and accepting\n\tTCP/IP connections on port 5432?\n"
        }
    ],
    "path": "$.args[0].args",
    "error": "Inconsistent object: connection error",
    "code": "invalid-configuration"
}

Same Error Message pops up

**Reply by TimReynolds100** [reply]
I get the same error using hasura3.  If you have resolved your issue can you share the solution?  Does Hasura have setup steps that are very specific and do not fail?  In lieu of that is there some way to support a setup like an advisor, support staff or consultant paid or not?  Is there some way to find a consultant that you might recommend?  Setup is way too frustrating and I suspect many potential clients just throw their hands in the air and walk away like I am about to do.  Onboarding should be a core competency not a frustrating disaster.
